www/filesender-config.js.php

```diff
@@ -47,6 +47,12 @@ function value_to_TF( $v )
     return json_encode( ($v ? true : false) );
 }
 
+$to = Config::get('transfer_options');
+$to_getalink = $to['get_a_link'];
+if ($to_getalink['available'] === false && $to_getalink['default'] === true) {
+  $get_a_link = true;
+}
+
```

Comment:
I suspect a more general way to deal with this may be in order, to expose all `transfer_options` (and `guest_options`) that have been set in config.php to be `available => false` AND `default => true` (i.e., they're active and _not_ exposed to the UI), since AFAICT they're all currently being unseen/ignored by the JavaScript code.

As it is this serves to validate the approach in one particular case (for the `get_a_link` transfer option) and fixes an existing bug #439. A new issue can be opened for the more general case in case more options from config.php need to be exposed.